content/en/tracing/profiler/profiler_troubleshooting.md

```diff
@@ -61,6 +61,20 @@ jdk.ObjectAllocationOutsideTLAB#enabled=true
 
 [Learn how to use override templates.](#creating-and-using-a-jfr-template-override-file)
 
+## Enabling the heap profiler
+
```

Comment:
If this is beta, we should add a statement about that here. Confirm and I will provide some wording.

Reply:
Yes, it's beta. Let's add a disclaimer.

Reply:
```suggestion
<div class="alert alert-info">The Java heap profiler feature is in beta.</div>
```